mobile/lib/util/environment.dart

```diff
@@ -26,6 +27,21 @@ class Environment {
         electrsEndpoint: electrsEndpoint,
         coordinatorPubkey: coordinatorPublicKey,
         p2PPort: lightningPort,
-        httpPort: httpPort);
+        httpPort: httpPort,
+        network: network);
+  }
+
+  static Network getNetwork() {
+    String network = const String.fromEnvironment('NETWORK', defaultValue: "regtest");
+    switch (network) {
+      case 'signet':
+        return Network.Signet;
+      case 'testnet':
+        return Network.Testnet;
+      case 'mainnet':
+        return Network.Bitcoin;
+      default:
+        return Network.Regtest;
+    }
```

Comment:
consider doing this part in rust (`from_str()`), and then we don't need to expose BDK enum at all.